doc/source/whatsnew/v1.1.0.rst

```diff
@@ -907,6 +907,8 @@ Groupby/resample/rolling
   to the input DataFrame is inconsistent. An internal heuristic to detect index mutation would behave differently for equal but not identical
   indices. In particular, the result index shape might change if a copy of the input would be returned.
   The behaviour now is consistent, independent of internal heuristics. (:issue:`31612`, :issue:`14927`, :issue:`13056`)
+- Bug in :meth:`SeriesGroupBy.agg` where any column name was accepted in the named aggregation of ``SeriesGroupBy`` previously.
```

Comment:
this should be 1 line  (e.g. remove the newline on line L910, sphinx will wrap it which is fine)

Reply:
I didnot know that. I'll change it.